py/mockbuild/plugin.py

```diff
@@ -15,13 +15,18 @@ def __init__(self, config, state):
         self.config = config
         self._hooks = {}
         self.state = state
+        
+        self.initted = False
 
         self.plugins = config['plugins']
         self.plugin_conf = config['plugin_conf']
         self.plugin_dir = config['plugin_dir']
 
     @traceLog()
     def init_plugins(self, buildroot):
+        if self.initted:
+            return
+        self.initted = True
```

Comment:
initted is not a word. Initialized is the correct word. And I would rather prefer something more descriptive. E.g. self.already_initialized